### ISSUE TITLE
Struggle running tensorboard

### ISSUE THREAD

**Issue by whiteGhost333** [Unity-Technologies/ml-agents#1619]

**Comment by whiteGhost333** [comment]
So, after training, I want to see the graphical results, etc. and I receive the following :

(ml-agents) C:\Users\msi>tensorboard --logdir=summaries
2019-01-21 11:46:59.095850: I T:\src\github\tensorflow\tensorflow\core\platform\cpu_feature_guard.cc:140] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2
TensorBoard 1.7.0 at http://DESKTOP-DVLHIQK:6006 (Press CTRL+C to quit)

This occurs after an apparently(no reason to think otherwise) successful training in 3D Ball Learning.

Thanks

**Comment by whiteGhost333** [comment]
so  the message about "not compiled to use AVX2 tensorboard" is to be expected? also , I had to ctrl + c to get to the command prompt after waiting several minutes.  Seems odd  perhaps?

**Comment by xiaomaogy** [comment]
Yes, the "not compiled to use AVX2 tensorboard" is expected and it is related to tensorflow, you can search on google and find the root reason for it. (One of the issue in tensorflow github explained it. )

Also you don't terminate it with ctrl+c if you want to use it. You keep it running so that you can view it from web browser.
